network-api/networkapi/wagtailpages/pagemodels/publications/article.py

```diff
@@ -113,6 +111,15 @@ class ArticlePage(FoundationMetadataPageMixin, Page):
         InlinePanel("footnotes", label="Footnotes"),
     ]
 
+    @property
+    def is_publication_article(self):
```

Comment:
Hi @KalobTaulien! 

Thanks in advance for checking out my PR, I was wondering if I can get some feedback on this function which I am using to tell the template that it is rendering an article page underneath a publication page? 

Right now I have it checking if any of the ancestors have the word "Publication" in them, however, I was concerned as I was not sure how likely we might possibly rename publication pages to something else in the future. 

Thanks again!